lib/fileHandler/index.js

```diff
@@ -47,6 +47,20 @@ const DEFAULT_UPLOAD_OPTIONS = {
 	queueSize: 4 //Ensures at least up to 1/4 of the RAM is utilised
 };
 
+const REQUIRED_FILE_PROPS = ['name', 'url', 'mime_type', 'expires']
+
+const isObjValid = fileObj => {
+    const fileObjKeyArr = _.keys(fileObj)
+    return REQUIRED_FILE_PROPS.every(reqField => fileObjKeyArr.includes(reqField))
+}
+
+const throwFileValidationErr = () => {
+    return when.reject({
+        code: '#connector_error',
+        message: 'The file object passed in must contain valid properties \'name\', \'url\', \'mime_type\', \'expires\'.'
```

Comment:
You don't need to do the backslash escaping if the outer single quotes were backticks instead.